src/components/skip/skip.scss

```diff
@@ -1,18 +1,23 @@
-.bottom-bar {
-  .btn-skip {
-    position: absolute;
+.btn-skip {
+  position: absolute;
+  right: 0;
+  border-radius: 4px;
+  background-color: rgba(0, 0, 0, 0.6);
+  color: #fff;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  font-size: 14px;
+  cursor: pointer;
+  padding: 7px;
+
+  &.interactive-area-position {
+    bottom: 0;
+  }
+
+  &.bottom-bar-position {
     top: 0;
-    right: 0;
     transform: translateY(-90%);
```

Comment:
90% is correct anyway?
if you hover the ui and the bottom bar is shown while the timeout, and once the timeout is done the element moved to the bottom bar, do you see any movement, or it's in the exact place? in any player size?

Reply:
Right is    -100% 
fixed